src/store/mutations.ts

```diff
@@ -162,24 +171,27 @@ export const mutations: MutationTree<CalChartState> = {
         const dot = currentSS.stuntSheetDots.find((dot) => dot.id === dotId);
         if (dot) {
           dot.dotTypeIndex = dotTypeIndex;
+          dot.calculateWarnings(state.selectedSS, dotId);
         }
       });
-      state.show.generateFlows(state.selectedSS);
```

Comment:
Why this removal? It seems like we would want to update the flows if updating dot types; I'm I missing something?

Reply:
Yes! Accidental removal...